Google.PowerShell/ReleaseFiles/PatchFromGcs.ps1

```diff
@@ -0,0 +1,43 @@
+﻿# Copyright 2016 Google Inc. All Rights Reserved.
+# Licensed under the Apache License Version 2.0.
+#
+# Updates Cloud Tools for PowerShell module to the latest found in Google Cloud Storage bucket g-cloudsharp-unsignedbinaries
```

Comment:
End sentences with a period. Also, we should line break script files at 80 cols or so.